CHANGELOG.md

```diff
@@ -1,5 +1,40 @@
 # Change Log
 
+## [3.4.00](https://github.com/kokkos/kokkos-kernels/tree/3.4.00) (2021-04-25)
+[Full Changelog](https://github.com/kokkos/kokkos-kernels/compare/3.3.01...3.4.00)
+
+**Features:**
+- SYCL: adding ETI and CMake logic for SYCL backend [\#924](https://github.com/kokkos/kokkos/pull/924)
```

Comment:
The link should really be: `https://github.com/kokkos/kokkos-kernels/pull/924`
The same is true for the other links below.

Reply:
Actually I just did that in the commit below.

Reply:
Ack, good catch, I reused the same scripting from kokkos but forgot to update the link for kokkos-kernels

Reply:
@lucbv looks like you beat me to the changelog repair, thanks for the fix!